src/unix/process.c

```diff
@@ -453,13 +453,13 @@ int uv_get_children_pid(pid_t ppid, uint32_t** proc_list, int* proc_count) {
   if (fp == NULL)
     return 127;
 
-  while (getline(&line, &len, fp) >= 0) {
+  while (fscanf(fp, "%u", &match_pid) > 0) {
```

Comment:
@eljefedelrodeodeljefe Not sure whether libuv does support any platform where `unsigned int != uint32_t`, but with *scanf, you may need to be more careful that pointer types match the format specifiers than with *printf (the worst case here would be overwriting part of another stack variable).

For `uint32_t` that should be `fscanf(fp, "%" SCNu32, &match_pid)`, I think.